examples/tools/tool-package-quickstart/my_tool_package/yamls/my_tool_package.tools.mlindex_tool.yml

```diff
@@ -0,0 +1,49 @@
+my_tool_package.tools.mlindex_tool.my_tool:
+  function: my_tool
+  inputs:
+    index_json:
+      type:
+      - string
+      generated_by_ux:
+        func_path: my_tool_package.tools.mlindex_tool.generate_index_json
+        func_kwargs: 
```

Comment:
The args can only be static value and can not refer flow inputs and node outputs.